image/cli/mascli/functions/install

```diff
@@ -174,6 +174,11 @@ function install_noninteractive() {
     export MAS_APP_CHANNEL_MONITOR=$1 && shift
     ;;
     --manage-channel)
+    echo
+    echo_h2 "Maximo IT Add-On"
+    echo -e "${COLOR_YELLOW} For information about your license, see https://ibm.biz/MAXIT81-License You are not required to accept the license terms unless you are installing Maximo IT add-on for Manage."
+    prompt_for_confirm_default_yes "Do you accept the license terms?" LICENSE_RESPONSE
```

Comment:
This is part of the noninteractive install so I am not sure you human would be expected to be prompted?

Reply:
Thanks Andrew, I removed this part.